test/conformance/ptp/ptp.go

```diff
@@ -992,3 +1055,248 @@ func printInterface(config ptpv1.PtpConfig, interfaceDetailsMap map[string]strin
 		}
 	}
 }
+
+func rebootSlaveNode(fullConfig testconfig.TestConfig) {
+	logrus.Info("Rebooting system starts ..............")
+
+	const (
+		daemonSetName    = "ptp-reboot"
+		containerName    = "container-00"
+		imageWithVersion = "quay.io/testnetworkfunction/debug-partner:latest"
```

Comment:
Is this image necessary here?  In my opinion qe should try to get required images in quay.io/openshift or openshift.io if at all possible, not elsewhere.

Reply:
@josephdrichard this is testframe pod used by cnf tests . currently it is at this repo. I don't think it will ever move  under opneshift. Since it is used only for e2e test , I think it is okay .